src/DonationForms/Actions/ConvertDonationFormBlocksToFieldsApi.php

```diff
@@ -110,7 +114,10 @@ protected function convertInnerBlockToNode(BlockModel $block, int $blockIndex)
         if ($node instanceof Node) {
             $node = $this->mapGenericBlockAttributesToNode($node, $block);
 
-            Hooks::doAction('givewp_donation_form_block_converted_to_node', $node, $block);
+            $this->blockNodeRelationships[$block->clientId] = [
+                'block' => $block,
+                'node' => $node,
+            ];
```

Comment:
@pauloiankoski I think we may need to move this to the `__invoke` function, as Sections are also nodes that can be shown/hidden, so we'd want to include them in the node relationships.

Reply:
@JasonTheAdams I didn't include sections on that list because although sections can be shown/hidden, they will always use a field node as a conditional. Also, we don't accept sections to be used as they can't be compared to a value. In other words, we don't need to add sections to that array as those will never be used.

Reply:
I can see why we didn't add it for FFM, but from the vantage point of the relationships, Sections are still a block being transformed into a node, so I think it's still worth keeping in the final relationships map.

Reply:
Makes sense.

Resolved: 04419f09c63000cfe19d45120c60a6dfdddd9a0b